packages/core/lib/patchers/aws_p.js

```diff
@@ -85,10 +85,17 @@ function captureAWSRequest(req) {
   }
 
   var traceId = parent.segment ? parent.segment.trace_id : parent.trace_id;
+  const data = parent.segment ? parent.segment.additionalTraceData : parent.additionalTraceData;
 
   var buildListener = function(req) {
-    req.httpRequest.headers['X-Amzn-Trace-Id'] = 'Root=' + traceId + ';Parent=' + subsegment.id +
+    let traceHeader = 'Root=' + traceId + ';Parent=' + subsegment.id +
```

Comment:
nit: maybe rename this variable to match the name below (`additionalTraceData`) for clarity on what this data is?

Reply:
This is not just the additional trace data, rather the complete trace header string, that's why the name `traceHeader`.